internal/ais/amss.go

```diff
@@ -0,0 +1,95 @@
+package ais
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+
+	"github.com/hashicorp/go-cleanhttp"
+)
+
+type AMSSClient struct {
```

Comment:
Why didn't you use https://github.com/artefactual-labs/ssclient-go?

Reply:
I took a look and I think it only covers a few endpoints from the `location` and `pipeline` namespaces, nothing from the `file` namespace. I thought you didn't use it in Enduro for the same reason.

Reply:
Ah! I honestly don't remember creating an AMSS API client in Enduro, are you sure it was me?  I can't even find an AMSS client in the Enduro code, but maybe I'm looking in the wrong place. :confused:

Reply:
Maybe it wasn't you. It's actually called `ssblob`, looks to be an abstraction for the AMSS location type:

https://github.com/artefactual-sdps/enduro/tree/main/internal/storage/ssblob

Reply:
Oh yeah, that was Keith and Jesús. :) I think the ssblob work predates https://github.com/artefactual-labs/ssclient-go.  It doesn't really matter if ssclient-go doesn't work with the endpoints you need.